### ISSUE TITLE
CORS policy: unable make api request to rawg api

### ISSUE THREAD

**Issue by apicgg** [mosh-hamedani/game-hub#2]
After deploying to vercel, the api request to rawg api is getting blocked due to CORS policy even after using correct api key.

Detail of the error - 
`Access to XMLHttpRequest at 'https://api.rawg.io/api/games?key=c7b18323a47d40c394ea5b019646b1f5' from origin 'https://game-hub-phi.vercel.app' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.`

**Comment by apicgg** [comment]
Hello @mosh-hamedani Have you encountered this issue with Vercel deployment? I have already created one PR regarding this. But now I am thinking it makes much more sense if we can address the issue into the project's README.md instead of changing the file `src/services/api-client.ts`. Let me know, I can create another PR for the same. Thanks. 🙂
